extensions/positron-python/package.nls.json

```diff
@@ -51,9 +51,9 @@
     "python.defaultInterpreterPath.description": "Path to default Python to use when extension loads up for the first time, no longer used once an interpreter is selected for the workspace. Must be an absolute path.",
-    "python.interpreters.include.markdownDescription": "List of folders (absolute paths) to search for Python installations. These folders are searched in addition to the default folders for your operating system.\n\nExample: if you have Python located at `/custom/pythons/3.10.4/bin/python`, add path `/custom/pythons` or `/custom/pythons/3.10.4` to this setting.\n\nNote:  If an interpreter is both included via `python.interpreters.include` and excluded via `python.interpreters.exclude`, it will not be displayed in the Positron UI.\n\nRequires a restart to take effect.",
-    "python.interpreters.exclude.markdownDescription": "List of interpreter paths or folders (absolute paths) to exclude from the available Python installations. These interpreters will not be displayed in the Positron UI.\n\nExample: Add `/usr/bin/python3` to exclude the specific interpreter, or `/opt/homebrew` to exclude all brew-installed Pythons on macOS.\n\nRequires a restart to take effect.",
-    "python.interpreters.override.markdownDescription": "List of absolute paths to Python executables or folders containing Python installations to override the list of the available Python installations. Only the interpreters found at the specified paths will be displayed in the Positron UI. This setting takes precedence over the `#python.interpreters.include#` and `#python.interpreters.exclude#` settings.\n\nExample: On Linux or Mac, add `/custom/pythons/3.10.4/bin/python` to include only this specific installation, or `/custom/pythons` to include only Python installations found within the directory.\n\nRequires a restart to take effect.",
+    "python.interpreters.include.markdownDescription": "List of absolute paths to Python executables or folders containing Python installations to include in interpreter discovery. These folders are searched in addition to the default folders for your operating system.\n\nExample: if you have Python located at `/custom/pythons/3.10.4/bin/python`, add path `/custom/pythons` or `/custom/pythons/3.10.4` to this setting.\n\nRequires a restart to take effect.",
+    "python.interpreters.exclude.markdownDescription": "List of absolute paths to Python executables or folders containing Python installations to exclude from the available Python interpreters. These interpreters will not be displayed in the Positron UI.\n\nIf an interpreter is both included via `#python.interpreters.include#` and excluded via `#python.interpreters.exclude#`, it will not be displayed in the Positron UI.\n\nExample: Add `/usr/bin/python3` to exclude the specific executable, or `/opt/homebrew` to exclude all brew-installed Python installations on macOS.\n\nRequires a restart to take effect.",
```

Comment:
> If an interpreter is both included via `#python.interpreters.include#` and excluded via `#python.interpreters.exclude#`, it will not be displayed in the Positron UI.

Would it make sense to add this in the include description as well?

Reply:
Referenced related settings here and in the R settings descriptions too 👍